src/w_zip.c

```diff
@@ -32,11 +32,13 @@ static void ConvertSlashes(char *path)
     }
 }
 
-static void AddWadInMem(mz_zip_archive *zip, const char *name, int index,
+static void AddWadInMem(w_handle_t handle, const char *name, int index,
                         size_t data_size)
 {
     I_Printf(VB_INFO, " - adding %s", name);
 
+    mz_zip_archive *zip = handle.p1.zip;
```

Comment:
I think we can omit the `.p1` part as long as all elements of all unions in the struct are named unambiguously.

Reply:
Anonimuos union is a GNU extension. MSVC didn't like it.